quoridor-logic-server/src/main/java/com/rl/mpquoridor/models/players/Player.java

```diff
@@ -0,0 +1,21 @@
+package com.rl.mpquoridor.models.players;
+
+import com.rl.mpquoridor.models.Pawn;
+import com.rl.mpquoridor.models.actions.TurnAction;
+import com.rl.mpquoridor.models.board.PhysicalBoard;
+import com.rl.mpquoridor.models.board.ReadOnlyPhysicalBoard;
+
+public abstract class Player {
+    protected ReadOnlyPhysicalBoard board;
+    public Player(ReadOnlyPhysicalBoard physicalBoard) {
+        this.board = physicalBoard;
+    }
+
+    public abstract void setPlayOrder(Pawn[] playOrder);
+    public abstract void setMyPawn(Pawn myPawn);
+    public abstract void illegalMovePlayed(String msg);
+    public abstract void triggerChange(); // todo: this signature could get Json or something.
+    // public abstract void triggerChange(JsonObject json);
```

Comment:
it will  get a GameEvent - the WebScoketManager will convert the Gson for you - or some adapter in between the classes.
The player class should not deal with Json or other formats